metabase/metabase
# [FE] "Combine columns" drill thru

Closes #39980

### Description
Adds combine column drill through to the chill mode header columns.


### How to verify

- Open the `Accounts` table in chill mode
- Click the header of a string column (eg. `Email`, `First Name` or `Plan`)
- Click `+ Combine columns`
- Play around with the column editor
- Click `Done`
- Verify the new column is added as the last column


### Demo

See video attached to [this Slack message](https://metaboat.slack.com/archives/C0645JP1W81/p1710427151050319)

<img width="361" alt="Screenshot 2024-04-19 at 14 52 22" src="https://github.com/metabase/metabase/assets/1250185/e57de922-897d-4446-8f23-8e77fe283435">
<img width="528" alt="Screenshot 2024-04-19 at 14 52 26" src="https://github.com/metabase/metabase/assets/1250185/d546f3df-efcd-49b7-bfbc-1df67b81b5d3">
<img width="502" alt="Screenshot 2024-04-19 at 14 52 34" src="https://github.com/metabase/metabase/assets/1250185/029e83a1-830e-483b-af46-f05dccf4cc21">
